src/RewriteHandler.ts

```diff
@@ -1,10 +1,13 @@
-import { FragmentDefinitionNode, parse, print } from 'graphql';
+import { ASTNode, FragmentDefinitionNode, parse, print } from 'graphql';
 import { extractPath, FragmentTracer, rewriteDoc, rewriteResultsAtPath } from './ast';
 import Rewriter, { Variables } from './rewriters/Rewriter';
 
 interface RewriterMatch {
   rewriter: Rewriter;
-  paths: ReadonlyArray<ReadonlyArray<string>>;
+  fieldPaths: ReadonlyArray<ReadonlyArray<string>>;
```

Comment:
Is this a breaking change to split `paths` into `fieldPaths` and `allPaths`?

Reply:
This isn't a breaking change. `paths` hasn't been split, it's just that the original variables has been renamed to be `fieldPath` and a new variable has been created called `allPath`. Just to make the distinction more clear.

Reply:
Ok, should be completely non breaking now.